README.md

```diff
@@ -104,7 +104,7 @@ composer require "openmage/magento-lts":"^19.4.0"
 composer require "openmage/magento-lts":"^20.0.0"
 ```
 
-<small>Note: be sure to select `y` if composer asks you to trust `magento-hackathon/magento-composer-installer`.</small>
+<small>Note: be sure to select `y` if composer asks you to trust `magento-hackathon/magento-composer-installer` or `cweagans/composer-patches`.</small>
```

Comment:
```suggestion
<small>Note: be sure to select `y` if composer asks you to trust `magento-hackathon/magento-composer-installer` and `cweagans/composer-patches`.</small>
```
?

Reply:
I think “or” is more correct. Especially since (depending on the OM version) that composer may not ask about cweagans/patches. “And” might imply something went wrong if the user doesn’t see it.